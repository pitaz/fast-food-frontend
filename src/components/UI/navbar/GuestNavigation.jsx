import React, {Fragment} from 'react';
import {Link } from 'react-router-dom';

const GuestNavigation = () => (
  <Fragment>
  <Link to="/signup">Sign Up</Link>
  </Fragment>
);

export default GuestNavigation;
