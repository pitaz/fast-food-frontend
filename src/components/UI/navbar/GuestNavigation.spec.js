/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import GuestNavigation from './GuestNavigation.jsx';


describe('GuestNavigation Component', () => {
  const wrapper = shallow(<GuestNavigation />);
  it('should render the GuestNavigation component', () => {

    expect(wrapper.exists()).toBe(true);
  });
  it('should match the snapshot', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
