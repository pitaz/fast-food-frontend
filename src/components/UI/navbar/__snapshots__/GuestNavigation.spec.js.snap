// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GuestNavigation Component should match the snapshot 1`] = `
<Fragment>
  <Link
    replace={false}
    to="/signin"
  >
    Sign In
  </Link>
  <Link
    replace={false}
    to="/signup"
  >
    Sign Up
  </Link>
</Fragment>
`;
